modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -25,14 +28,46 @@ The `high-limit` format allows more nodes and relationships to be stored than `s
 
 You can either set the store format when creating a new database or change the store format of an existing database.
 
+The default store format of new databases is `aligned` and is controlled by the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration setting in the _neo4j.conf_ file.
+
 === Creating new databases
 
-The default store format of new databases is controlled by the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration setting in the _neo4j.conf_ file.
-Or the store format can be passed as an argument to the tool creating the database, e.g. xref:tools/neo4j-admin/neo4j-admin-import.adoc#import-tool-full[`neo4j-admin database import`] or xref:backup-restore/copy-database.adoc[`neo4j-admin database copy`] commands.
+There are several ways to create a new database in a specific store format:
+
+* Specify the store format when creating a new database using the `CREATE DATABASE` Cypher statement.
+For example:
++
+[source,cypher]
+----
+CREATE DATABASE blockdb OPTIONS {storeFormat: 'block'}
```

Comment:
This is still not supported official syntax https://neo4j.com/docs/operations-manual/current/database-administration/standard-databases/manage-databases/#manage-databases-create-database-options
It should not be in the official documentation

Reply:
So, it seems that David Pond's blog post is not correct, which by the way is about the beta version of the block format.